tracer/build/PackageVersionsGeneratorDefinitions.json

```
@@ -1,4 +1,11 @@
 [
+  {
+    "IntegrationName": "AwsLambda",
+    "SampleProjectName": "Samples.AWS.Lambda",
+    "NugetPackageSearchName": "Amazon.Lambda.Core",
+    "MinVersion": "2.1.0",
+    "MaxVersionExclusive": "2.1.1"
```

The purpose of this file is to ensure we automatically test against _new_ versions of the NuGet package when they come out. Given the integration doesn't seem to have any restrictions, we should probably test against all 2.x versions at least?
```suggestion
    "MaxVersionExclusive": "3.0.0"
```

---

I have actually reverted this because I didn't correctly understand what it was.
We are NOT testing Amazon.Lambda.Core but the customer handler. It has nothing to do with Amazon.Lambda.Core

Yes this is need because unlike all other integrations tests :
- a docker container (AWS Lambda based) is built and the Sample App is copied into it
- the newly built tracer is copied into that container as well
- we run the test (aka the Sample app) inside this container (the tracer agent is also inside)
- so we need a way to tell to which port traces should be flushed out from the Lambda Container to the container which actually run the Linux integration tests

That's also why we need a fixed port, so the container knows in advance to which port he should forward the traces that's why we set this in our Lambda container
`environment: DD_TRACE_AGENT_URL=http://integrationtests:5002`